src/cpp/dynamic-types/TypesBase.cpp

```diff
@@ -16,64 +16,69 @@
  * @file TypesBase.cpp
  */
 
-#include <fastrtps/types/TypesBase.h>
 #include <fastcdr/Cdr.h>
+#include <fastcdr/CdrSizeCalculator.hpp>
+#include <fastrtps/types/TypesBase.h>
 
 namespace eprosima {
-namespace fastrtps {
-
-using namespace rtps;
-
-namespace types {
-
-void MemberFlag::serialize(
-        eprosima::fastcdr::Cdr& cdr) const
+namespace fastcdr {
+template<>
+size_t calculate_serialized_size(
```

Comment:
`MemberFlag` and `TypeFlag` are bitmasks and now are not generated into a structure. So I think these methods should be removed (at least Fast DDS-Gen does not generate auxiliary code with the [XTypesBase.idl](https://github.com/eProsima/Fast-DDS/blob/bugfix/type-object/builtin-annotations/include/fastrtps/types/XTypesBase.idl)